src/storage/controllers/storageController.ts

```diff
@@ -0,0 +1,23 @@
+import { Logger } from '@map-colonies/js-logger';
+import { RequestHandler } from 'express';
+import httpStatus from 'http-status-codes';
+import { injectable, inject } from 'tsyringe';
+import { SERVICES } from '../../common/constants';
+import { StorageManager } from '../models/storageManager';
+import { IStorageStatusResponse } from '../../common/interfaces'
+
+type GetTaskByJobIdHandler = RequestHandler<{ jobId: string }, IStorageStatusResponse, string>;
```

Comment:
the name of the type is confusing, mabye i missing something but I think the name should be relative to the issue of storage

Reply:
My mistake, renamed.